sigstore/_verify.py

```diff
@@ -136,6 +138,13 @@ def verify(
 
         `expected_cert_oidc_issuer` is the expected OIDC Issuer Extension within `certificate`.
 
+        `offline_rekor_entry` is an optional offline `RekorEntry` to verify against. If supplied,
+        verification will be done against this entry rather than the against the online
+        transparency log. Offline Rekor entries do not carry their Merkle inclusion
+        proofs, and as such are verified only against their Signed Entry Timestamps.
+        This is a slightly weaker verification verification mode, as it does not guarantee
+        the log's consistency.
```

Comment:
nit: Even with an online lookup for inclusion, there's no guarantee of consistency, unless a consistency proof is requested and verified against a previously persisted checkpoint/signed root hash. To be more precise, I'd say that "it does not demonstrate inclusion in the log" (I'm using demonstrate instead of guarantee since technically to guarantee it, you'd need witnessing too)